lib/fluent/plugin/monitoring.rb

```diff
@@ -97,21 +99,14 @@ def initialize(project_id, monitored_resource, gcm_service_address)
       require 'opencensus'
       require 'opencensus-stackdriver'
       @log = $log # rubocop:disable Style/GlobalVars
-      @recorder = OpenCensus::Stats.ensure_recorder
-      @exporter = OpenCensus::Stats::Exporters::Stackdriver.new(
-        project_id: project_id,
-        metric_prefix: 'agent.googleapis.com/agent',
-        resource_type: monitored_resource.type,
-        resource_labels: monitored_resource.labels,
-        gcm_service_address: gcm_service_address
-      )
-      OpenCensus.configure do |c|
```

Comment:
Tested that opencensus ingestion still works:

![image](https://user-images.githubusercontent.com/5287526/93401677-bc648d80-f850-11ea-816a-d2a5e211fe16.png)